docs/source/reference/modding/extended.rst

```diff
@@ -0,0 +1,211 @@
+
+1. ``Idle`` (with ``Start`` and ``Stop`` keys)
+
+2. ``ContainerOpen`` (with ``Start``, ``Loot`` and ``Stop`` keys)
+
+3. ``ContainerClose`` (with ``Start`` and ``Stop`` keys)
+
+The ``Idle`` group can have the zero time, the ``Loot`` key for ``ContainerOpen`` allows to play a part of opening animation in the background.
+
+For example, with the following setup, the opening animation has 1.0 sec duration and shows the container window in the middle of opening animation:
+
+::
+
+    0.0: ContainerOpen: start
+    0.5: ContainerOpen: loot
+    1.0: ContainerOpen: stop
+
+It is also possible to attach opening/closing sounds to container's animations:
+
+::
+
+    1.0: ContainerClose: open
+    1.01: Sound: AC_dw_drawer_close
+    2.0: ContainerClose: stop
+
+An approach is the same as for other animation sounds (you need to register sound record in the ESM/ESP file).
+Note that the sound starting time value is slightly higher than the closing animation start, otherwise sound will be played at the end of opening animation in this example.
+
+It is important to assign a RootCollisionNode to the container mesh -- the collision shape will depend on the animation state otherwise, what can have a performance impact.
+
+Advantages of described approach over old animated containers mods:
+
+1. There is no need to attach a script to every container
+
+2. Part of opening animation can be played in the background, so we do not waste player's time
+
+An example of mod which uses this feature is `OpenMW Containers Animated`_.
+
+
+Day/night state support
+-----------------------
+
+It is possible to add one or more NiSwitchNodes named ``NightDaySwitch``.
+Every such switch should have at least two child nodes
+(first one represents the node in the normal mode, second one - in the night mode, optional third node - a day mode for interiors).
+
+The behavior of such a model:
+
+1. During day in exteriors it is in the "common" mode (child 0).
+
+2. During night in exteriors it is in the "night" mode (child 1).
+
+3. During day in interiors it is in the "common" mode or "interior day" mode (child 2) depending on weather.
+
+4. During night in interiors it is in the "common" mode.
+
+An actual switch time depends on weather settings in openmw.cfg:
+
+::
+
+    fallback=Weather_Sunrise_Time,6
+    fallback=Weather_Sunset_Time,18
+    fallback=Weather_Sunrise_Duration,2
+    fallback=Weather_Sunset_Duration,2
+
+With this example a night mode starts at 20:00 and ends at 6:00.
+
+In interiors we check if the weather is bright to enter the "interior day" mode using the Glare_View setting. If it is >= 0.5, we consider weather to be bright:
+
+::
+
+    fallback=Weather_Clear_Glare_View,1
+    fallback=Weather_Foggy_Glare_View,0.25
+
+With this example we use the "interior day" mode for Clear weather, but do not use it for Foggy weather.
```

Comment:
With these settings "interior day" mode would be used for Clear weather but would not be used for Foggy weather.